ffmpeg_plugin/README.md

```diff
@@ -3,56 +3,70 @@
 These instructions build a version of ffmpeg with the SVT-HEVC plugin.  The steps differ depending
 on whether using a Linux or Windows build machine.
 
-For Linux:
-1. Build and install SVT-HEVC 
+## For Linux:
+1\. Build and install SVT-HEVC 
+``` bash
 - git clone https://github.com/OpenVisualCloud/SVT-HEVC
 - cd SVT-HEVC/Build/linux
 - ./build.sh debug|release
 - cd debug|release
 - make install
 - export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/usr/local/lib
 - export PKG_CONFIG_PATH=${PKG_CONFIG_PATH}:/usr/local/lib/pkgconfig
-
-For Windows:
-0. Setup environment
+```
+## For Windows:
+0\. Setup environment
+``` bash
 - Install Visual Studio 2017
 - Install msys2-x86_64 to C:\msys64 following instructions found on http://www.msys2.org/
 - Run "C:\msys64\mingw64.exe" to open a msys2 mingw-w64 shell.  All subsequent build steps should be done in this shell.
 - Execute "pacman -S --needed make mingw-w64-x86_64-cmake mingw-w64-x86_64-gcc mingw-w64-x86_64-yasm mingw-w64-x86_64-SDL2 perl diffutils pkg-config git tar" in the console
-
-1. Build and install SVT-HEVC
+```
```

Comment:
you might not want to use the code block for this one since it is not actually supposed to be run in bash